src/containers/RegistrationContainer.js

```diff
@@ -41,6 +41,15 @@ const RegistrationContainer = props => {
   const [img, setImg] = useState(null)
   const [interests, setInterests] = useState([])
   const [nextButtonActive, setNextButtonActive] = useState(true)
+  const [username, setUsername] = useState('')
+
```

Comment:
Also here I feel this is unnecessary, since user should  already be available in state. Why not just update the username together with nickname in a similar manner?

Reply:
Done.